CIP-1694/README.md

```diff
@@ -0,0 +1,504 @@
+
+In contrast to other protocol parameter updates, hard forks (or, currently, changes to the protocol's major version) require much more attention.
+Indeed, while other protocol parameter changes can be performed without significant software changes,
+a hard fork assumes that a supermajority of the network has upgraded the node to support the new set of features that are introduced by the upgrade. This means that the timing of a hard fork event must be communicated well ahead of time to all Cardano users, and requires coordination between stake pool operators, wallet providers, DApp developers, and the node release team.
+
+Hence, this proposal promotes hard fork initiations as a standalone governance action, separate from protocol parameter updates, as in the Shelley scheme.
+
+### Treasury Withdrawals vs Project Catalyst
+
+Project Catalyst is currently one of the main drivers for withdrawals from the Cardano treasury. Each Catalyst round is usually followed by hundreds - if not thousands - of MIR requests to deliver funding to selected projects. In this new governance framework, however, we will only permit one trasury withdrawal per epoch.
+
+Since all withdrawal requests from the treasury must fit into a single transaction, this limits the number of projects that can be funded in a single epoch. If necessary, this can be worked around by e.g. splitting funding over multiple epochs, by transferring funds to a temporary holding pot, or by restricting the number of projects that are funded in each round.
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ] A new ledger era is enabled on the Cardano mainnet, which implements the above specification. This will be split into two stages, as described above.
+
+### Implementation Plan
+
+This proposal restricts itself to the core on-chain mechanism. However, several issues remain to be discussed. These include:
+
+1. off-chain DRep expectations;
+2. bootstrapping the Constitutional Committee;
+3. the initial Constitution;
+4. DRep incentives;
+5. which values in the [ratification requirement table](#requirements) should be hard-coded, and which should be protocol parameters?
+
+Some of these issues may be addressed by changes to this document; others will need to be addressed in new CIPs or discussion documents.
+
+#### New Protocol Parameters
+
+New protocol parameters will be needed for the following:
+
+* the governance action deposit amount
+* governance action expiration (as a number of epochs from the current epoch)
+* thresholds for treasury withdrawals
+* each of the ratification thresholds
+
+As described above, some  of these will be updatable; others will be hard coded.
+
+// TODO: Decide on the initial parameter values and whether they should be updatable.
+
+In addition, the initial Constitutional Committee and the initial Constitution will need to be defined as part of a bootstrap process.
+
+#### Changes to the existing ledger rules
```

Comment:
We did not do such a thing for the three CIPs in the Babbage ledger era. In fact, we didn't even mention the specific ledger rule changes. I'm open to suggestions!

Reply:
Well, that seems to me to be a deficiency: perhaps the ideal ledger CIP would explicitly include a full specification of the changes to the ledger rules :shrug: at any rate, since you _have_ included material about the changes to the ledger rules, it seems like it's part of the specification!

Reply:
I am happy to arrange it however the CIP boards deems best! (btw, is the board listed in the main README still accurate?)

Reply:
yes, it's currently accurate as posted here: https://github.com/cardano-foundation/CIPs#current-editors